store.go

```diff
@@ -26,12 +26,20 @@ var (
 	ErrTraceNotFound = errors.New("trace not found")
 )
 
+type TracesOpts struct {
+	Timespan Timespan
+}
+
+func DefaultTracesOpts() TracesOpts {
+	return TracesOpts{}
+}
+
 // A Queryer indexes spans and makes them queryable.
 type Queryer interface {
 	// Traces returns an implementation-defined list of traces. It is
 	// a placeholder method that will be removed when other, more
 	// useful methods are added to Queryer.
-	Traces() ([]*Trace, error)
+	Traces(opts TracesOpts) ([]*Trace, error)
```

Comment:
We should update the docs here to say something like:

```
Traces returns an implementation-defined list of traces according to the options.
```

Since it's not a placeholder method anymore.